lib/jmap/mail/extensions/public_asset/get/get_public_asset_method.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/id_converter.dart';
+import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart';
+import 'package:jmap_dart_client/jmap/core/method/method.dart';
+import 'package:jmap_dart_client/jmap/core/method/request/get_method.dart';
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'get_public_asset_method.g.dart';
+
+@JsonSerializable(converters: [AccountIdConverter(), IdConverter()])
```

Comment:
how about to use nullable converter ?

Reply:
```dart
mixin OptionalIds {
  @JsonKey(includeIfNull: false)
  Set<Id>? ids;

  void addIds(Set<Id> values) {
    if (ids == null) {
      ids = Set();
    }
    ids?.addAll(values);
  }
}
```

`Id` in `OptionalIds` is not null, so we cannot use `IdNullableConverter` there.